tests/acceptance-tests/lib/basic-user-actions.js

```diff
@@ -3,11 +3,13 @@ const yaml = require("js-yaml");
 const fs   = require("fs");
 
 const eleMap = yaml.safeLoad(fs.readFileSync("./tests/acceptance-tests/elements/element-map.yml", "utf8"));
+const usrData = yaml.safeLoad(fs.readFileSync("./tests/acceptance-tests/config/user-data.yml", "utf8"));
+const browserConfig = yaml.safeLoad(fs.readFileSync("./tests/acceptance-tests/config/settings.yml", "utf8"));
+
```

Comment:
Can we just convert these either to JSON or just JS Objects to eliminate the need to load YAML?